src/recipe/Baked.Recipe.Service.Application/Database/PostgreSql/PostgreSqlDatabaseFeature.cs

```diff
@@ -15,13 +16,17 @@ public void Configure(LayerConfigurator configurator)
             services.AddSingleton<ITransaction, FlatTransaction>();
         });
 
+        configurator.ConfigureFluentConfiguration(fluent =>
+        {
+            if (_autoUpdateSchema)
+            {
+                fluent.ExposeConfiguration(c => new SchemaUpdate(c).Execute(false, true));
+            }
```

Comment:
why not provide a helper for this as well :thinking: ? you did it in MaxFetchDepth etc.